lightspeed_stack_providers/providers/inline/safety/lightspeed_question_validity/safety.py

```diff
@@ -48,11 +48,11 @@ async def run_shield(
         self,
         shield_id: str,
         messages: list[Message],
-        params: dict[str, Any] = None,
+        params: dict[str, Any] | None = None,
     ) -> RunShieldResponse:
-        shield = await self.shield_store.get_shield(shield_id)
-        if not shield:
-            raise ValueError(f"Unknown shield {shield_id}")
+        # shield = await self.shield_store.get_shield(shield_id)
+        # if not shield:
+        #     raise ValueError(f"Unknown shield {shield_id}")
```

Comment:
Why are we changing question_validity shield here ? Do we want to commit this change ?